Cargo.toml

```diff
@@ -31,11 +31,11 @@ harness = false
 name = "benches"
 
 [dependencies]
-log = "0.3.9"
-slog = "2.2"
+log = { version = "0.3", features = ["release_max_level_debug"] }
```

Comment:
if we really want to use trace, how can we use now?

Reply:
Use debug version, and there will be trace logs.

Reply:
Don't we build in debug for release?

Reply:
From now on, as far as I know, we haven't used debug in production.

Reply:
debug version is too slow, so it is mainly used in dev, instead of prod. @Hoverbear And when we meet issues, we mainly trace it by logs (>= info) and metrics.

Reply:
Hm, ok. I was referring to https://github.com/tikv/tikv/blob/master/Cargo.toml#L140, but perhaps it's something else.